src/js/base/editing/History.js

```diff
@@ -11,6 +11,12 @@ define(['summernote/base/core/range'], function (range) {
 
     var makeSnapshot = function () {
       var rng = range.create(editable);
+      var commonAncestor = rng.commonAncestor() || (rng.sc.isEqualNode(rng.ec)) ? rng.sc : null;
```

Comment:
I'm not a huge fan of the latter part of the above line, as I feel a DOM utility like `commonAncestor` should return the element if both are identical instead of `null`. However, I'm leery of changing a low-level utility like that, so figured it's better to address it up here.

Really, this is just an alternative to the next line's conditional being `!(editable.contains(rng.sc) && editable.contains(rng.ec))`. Seemed more logical to compute the common ancestor and ensure that single element exists within editable container.